src/components/PrintForm/Serializer/SHOGunMapFishPrintV3WMSSerializer.ts

```diff
@@ -23,6 +23,10 @@ export class SHOGunMapFishPrintV3WMSSerializer extends MapFishPrintV3WMSSerializ
       serialized.baseURL = `http://shogun-geoserver:8080${serialized.baseURL}`;
     }
 
+    if (serialized.baseURL.startsWith('/geoserver/SHOGUN')) {
+      serialized.baseURL = `http://shogun-geoserver:8080${serialized.baseURL}`;
```

Comment:
Even if the value is not intended to be changed, we might want to move the base url to the external config file.

Reply:
Yes thank you for the idea @dnlkoch i will fix this in a follow up